components/alert/alert.css

```diff
@@ -0,0 +1,54 @@
+
+
+:root {
+  --border-color: rgb(0, 0, 0);
+  --danger-bg-color: #b80c00;
+  --warning-bg-color: rgb(233, 115, 18);
+  --success-bg-color: rgba(144, 221, 99, 0.671);
+  --danger-color: #F50F00;
+  --warning-color: rgb(230, 217, 198);
+  --success-color: rgb(31, 187, 44);
+}
```

Comment:
you can move this root element to component.css

Reply:
ok will do that

Reply:
just made changes. Have added the css variables to a designated folder and linked them.